src/plugins/data/public/types.ts

```diff
@@ -28,7 +28,8 @@ import { createFiltersFromRangeSelectAction, createFiltersFromValueClickAction }
 import { ISearchSetup, ISearchStart, SearchEnhancements } from './search';
 import { QuerySetup, QueryStart } from './query';
 import { IndexPatternsContract } from './index_patterns';
-import { IndexPatternSelectProps, StatefulSearchBarProps } from './ui';
+import { StatefulSearchBarProps } from './ui';
+import { IndexPatternSelectProps } from './ui/index_pattern_select';
```

Comment:
nit: `IndexPatternSelectProps` should still be exported from the `ui/index` file

Reply:
`IndexPatternSelectProps` is exported from the `ui/index` file. Let me clean up this import statement.